packages/bsky/src/api/app/bsky/actor/getProfiles.ts

```diff
@@ -13,11 +13,11 @@ import { createPipeline, noRules } from '../../../../pipeline'
 export default function (server: Server, ctx: AppContext) {
   const getProfile = createPipeline(skeleton, hydration, noRules, presentation)
   server.app.bsky.actor.getProfiles({
-    auth: ctx.authOptionalVerifier,
+    auth: ctx.authVerifier.standardOptional,
     handler: async ({ auth, params, res }) => {
       const db = ctx.db.getReplica()
       const actorService = ctx.services.actor(db)
-      const viewer = auth.credentials.did
+      const viewer = auth.credentials.iss
```

Comment:
Wondering in the back of my head if we should be looking at `sub` 🤔 I suppose it's known/implied that for this kind of credential `iss` and `sub` are pretty much the same, but I could imagine an implementation looking to set `iss` to the source pds.

Reply:
In any case, it's not super specific to this work.  We can reconsider later, just didn't want to lose the thought.